### DISCUSSION TITLE
[Bug]: Better cargo splitting algorithm?

### DISCUSSION THREAD

**Discussion by NastyFlytrap** [OpenTTD/OpenTTD#11723] [General]
### Version of OpenTTD

13.4 Win10 60bit

### Expected result

Cargo being split fairly between competing stations

### Actual result

Competitor puts down one road depo, has 2-4 trucks run between there, and the game gives them 1300 crates of goods to transport, meanwhile my trains that could actually transport that much, get like 200-400.

_**When they put the station down, my ratings were higher than theirs.**_ 
They arent now, because all my trains do is sit around and wait for cargo they are not going to get, and in either scenario the game keeps giving cargo to them and not me. 

I've hated this since the original transport tycoon and if i were to replace an aspect of the original game for the better, this would definitely be the first thing because fuck this. 

### Steps to reproduce

Have station transport cargo
Have competitor place their station
Lose

**Comment by krysclarke** [comment]
I doubt this is actually a bug in the game, as it would definitely have been reported before now if it's been an issue for as long as you say it has been.

In any case, to assist with diagnosing this issue further, please provide a save game where you're experiencing this issue.

**Comment by ldpl** [comment]
First of all, this isn't how it currently works. All things equal, station with vehicles waiting will be getting better ratings and more cargo. If yours doesn't that means your opponent has some advantages like statue in the town or newer vehicles. Also, amount of cargo waiting on station is not the same as amount given to it. 
In this situation both companies should be receiving about the same amount of cargo. Which may not be particularly fair, but then how would game measure this "fairness"? It's not like the game doesn't try to weight in some factors for better distribution, there just aren't any useful things to measure.
And the game already has some ways for mods to control station ratings and cargo distribution. But I'm yet to see any mod do complex cargo distribution strategies. Only simple things like disabling any competition on secondary industries, which is by far the most significant problem in multiplayer wrt cargo distribution.

**Comment by TrueBrain** [comment]
Although I can understand it is frustrating to lose to the competitor, as explains by @ldpl , it is not an actual bug. But as I think it would be valuable to have a conversation, I moved this to our Discussion forum.
